apps/payment/migrations/0031_fikensale_description.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.1.9 on 2019-08-22 16:57
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('payment', '0030_fikensale'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='fikensale',
+            name='description',
+            field=models.CharField(default='TEST', max_length=200),
```

Comment:
This doesn't look useful :p I'd rather have an empty string than a "test" string.

Reply:
Did not really see that it was added

Reply:
:'D